app/models/event.rb

```diff
@@ -1,4 +1,8 @@
 class Event < ApplicationRecord
+
+  GUIDELINE_MESSAGE = 'Guidelines must be defined before event can be opened.'
```

Comment:
I like this cleaning up of duplication but I wonder if we shouldn't use a bundle for this?  Also, incomplete_checklist_items is using some of these messages as well.